core/src/main/java/com/novoda/noplayer/SubtitleView.java

```diff
@@ -31,12 +30,12 @@ public SubtitleView(Context context, AttributeSet attrs) {
     }
 
     public void setCues(TextCues textCues) {
```

Comment:
Can this `textCues` ever be null? Because if it can, line 37 can throw `NullPointerException`.